core/tests/karma.conf.js

```diff
@@ -8,7 +8,8 @@ if (isMinificationNeeded) {
 module.exports = function(config) {
   config.set({
     basePath: '../../',
-    frameworks: ['jasmine'],
+    frameworks: ['jasmine-jquery', 'jasmine'],
+    // jasmine-jquery is used to load contents of external JSON files in tests.
```

Comment:
Ah, sorry, one nit. Comments should always go _above_ the lines they refer to, not below. Could you please fix that?